SphereEversion/ToMathlib/Algebra/Ring/Periodic.lean

```diff
@@ -97,8 +97,6 @@ theorem continuous_proj𝕊₁ : Continuous proj𝕊₁ := continuous_quotient_m
 
 theorem isOpenMap_proj𝕊₁ : IsOpenMap proj𝕊₁ := QuotientAddGroup.isOpenMap_coe
 
-attribute [fun_prop] Continuous.prod_map
```

Comment:
This attribute already existed, right?

Reply:
It does now in mathlib; I didn't check if it did before.